sdk/src/claim.rs

```diff
@@ -737,13 +737,12 @@ impl Claim {
         }
     }
 
-    /// Not ready for use!!!!!
```

Comment:
This _appears_ to be ready for as the tests I've been running on the redactions work well. I could be missing something, however.